### DISCUSSION TITLE
Campus Experts for teachers (or such)

### DISCUSSION THREAD

**Discussion by proflivio** [community/community#103143] [GitHub Education]
### Hi everyone,

Hi,
is there a program like Campus Experts for teachers?
I am a CS teacher (verified here as faculty), aspiring to build a community of fellow teachers for support and personal developement, but also for collaboratively making lessons and activities for their students, maybe even programming some apps for solving teacher's problems (but that is too ambitious for now :). It would be great to involve future teachers, maybe some gifted pupils as well.

I wouldn't know how to start building such a community at the moment, and since I cannot apply for the Campus Expert program, are there resources that could help me with that? Are the repos of previous Campus Expert batches free to peruse? (I found one in the wild, but it's 7 years old.)

**Comment by gil-son** [comment]
Hello!

In this page from [Github Education](https://education.github.com/experts), you can find it

**Reply by proflivio** [reply]
Thanks for answering, but the Campus Experts program is for students with the Student Developer Pack. I'm not a student, I am a teacher verified as faculty.

I'm not asking to plant a flag or be certified in any way, I'm just asking if there are (similar) resources from which I can learn how to build and manage a (small) community.

**Reply by mishmanners** [reply]
[Please go to the teacher page](https://education.github.com/teachers) and you'll see all the benefits for teachers as well as a button to apply for the Teacher benefits such as GitHub Classroom.
